quipucords/api/common/util.py

```diff
@@ -45,6 +45,28 @@ def convert_to_int(value):
     return int(value)
 
 
+def is_boolean(value):
+    """Check if a value is a bool cast as string.
+
+    :param value: The value to check
+    :returns True if it is a bool, False if not
+    """
+    if value == 'True' or value == 'False':
+        return True
+    return False
+
+
+def convert_to_boolean(value):
```

Comment:
Python should do this with bool()

Reply:
Actually, sounds like bool does the wrong thing here.  This suggestion seems reasonable instead. https://stackoverflow.com/questions/715417/converting-from-a-string-to-boolean-in-python  May want to check if you are dealing with a string first.